Apps/Src/Task_LogInfo.c

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+#include "os.h"
+#include "EEPROM.h"
+#include "Charge.h"
+#include "Tasks.h"
+
+void Task_LogInfo(void *p_arg) {
```

Comment:
How large is our EEPROM? This seems like a quick way to completely wear out the flash, or at least use up our memory space.

Wouldn't this be better to just do over CAN?

Reply:
The EEPROM is 16 KB, and supports 4 million write cycles, which I think is enough for us to run this for several months.

There isn't really a good way to do this over CAN because we need to reload the state of charge from the EEPROM on start up, since the RAM will be wiped when the BPS powers off.